docs/data/date-pickers/migration-lab/migration-lab.md

```diff
@@ -0,0 +1,85 @@
+---
+title: MUI X - Migration from the lab
+---
+
+# MUI X - Migration from the lab
+
+<p class="description">MUI Date and time pickers are now available on X!</p>
+
+## Introduction
+
+This is a reference for migrating your site's pickers from `@mui/lab` to `@mui/x-date-pickers` or `@mui/x-date-pickers-pro`.
+This migration should be easy
+
+We explored the reasons of this migration in a [blog post](https://mui.com/blog/lab-date-pickers-to-mui-x/)
+
+## License
+
+Most of our components remains MIT and are accessible for free in `@mui/x-date-pickers`.
+
+The components `DateRangePicker`, `DateRangePickerDay`, `DesktopDateRangePicker`, `MobileDateRangePicker` and `StaticDateRangePicker`
+were marked as "intended for MUI X Pro" in our documentation and are now part of MUI X Pro.
+
+If you are using one of these components, you will have to take a Pro license in order to migrate to `@mui/x-date-pickers-pro` (see the [Pricing](https://mui.com/pricing/) page for more information).
+
+**Note**: If you already have a license for `@mui/x-data-grid-pro`, you can use the same one for `@mui/x-date-pickers-pro` with no additional fee!
+
+## Migration steps
+
+### 1. Install MUI X packages
+
+#### Community Plan
+
+```sh
+// with npm
+npm install @mui/x-date-pickers
+
+// with yarn
+yarn add @mui/x-data-grid
+```
+
+#### Pro Plan
+
+```sh
+// with npm
+npm install @mui/x-date-pickers-pro @mui/x-license-pro
+
+// with yarn
+yarn add @mui/x-date-pickers-pro @mui/x-license-pro
+```
+
+When you purchase a commercial license, you'll receive a license key by email.
+You must set the license key before rendering the first component.
+
+```jsx
+import { LicenseInfo } from '@mui/x-license-pro';
+
+LicenseInfo.setLicenseKey(
+  'x0jTPl0USVkVZV0SsMjM1kDNyADM5cjM2ETPZJVSQhVRsIDN0YTM6IVREJ1T0b9586ef25c9853decfa7709eee27a1e',
+);
+```
+
+More information [here](/x/advanced-components/#license-key-installation)
+
+### 2. Run the code mod
+
+We have prepared a codemod to help you migrate your codebase
+
+```sh
+npx @mui/codemod v5.0.0/date-pickers-moved-to-x <path>
+```
+
+Which will transform the imports like this:
+
+```ts
+// before
+import DatePicker from '@mui/lab/DatePicker';
+import DateRangePicker from '@mui/lab/DateRangePicker';
+import { DatePicker, DateRangePicker } from '@mui/lab';
+
+// after
+import DatePicker from '@mui/x-date-pickers/DatePicker';
+import DateRangePicker from '@mui/x-date-pickers-pro/DateRangePicker';
+import { DatePicker } from '@mui/x-date-pickers'; // DatePicker is also available in `@mui/x-date-pickers-pro`
+import { DateRangePicker } from '@mui/x-date-pickers-pro';
```

Comment:
I have two questions about the link between community and pro.

- If it pro package export everything. It could be nice to have a sentence on top explaining that pro users do not need to add the community package

Does the following line works?
```jsx
import DatePicker from '@mui/x-date-pickers/DatePicker';
```

Because in the doc examples, we use

```jsx
import { DatePicker } from '@mui/x-date-pickers/DatePicker';
```
and when I remove braces, I get the following message

> Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
> Check the render method of `BasicDatePicker`.

Reply:
```ts
import DatePicker from '@mui/x-date-pickers/DatePicker';
```

No we don't support default export, I'll fix it

Reply:
> If it pro package export everything. It could be nice to have a sentence on top explaining that pro users do not need to add the community package


Done